fastlane/Fastfile

```diff
@@ -0,0 +1,20 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation on https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Run all unit tests for ImagineEngine (best engine)"
+  lane :tests do
+    run_tests(scheme: "ImagineEngine-iOS")
+    run_tests(scheme: "ImagineEngine-macOS")
+    run_tests(scheme: "ImagineEngine-tvOS")
+
+    danger
```

Comment:
Look at how beautiful and easy to maintain this is ❤️

Reply:
uses the new fancy terms instead of the secret in-club things boarding or whatever this was called before

Reply:
Haha, true story, I still like the old names though. We need to rename `danger` to `verify_the_changes_are_introduced_the_right_way_tm_copyright_by_orta_manager`

Reply:
haha - yah right